microsoft-edge/extensions-chromium/samples.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Samples for Microsoft Edge extensions
+description: Samples for Microsoft Edge extensions.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: devtools
+ms.date: 01/27/2023
+---
+# Samples for Microsoft Edge extensions
+
+Use the demos in the [MicrosoftEdge / Demos](https://github.com/MicrosoftEdge/Demos#readme) repo to learn how to use Microsoft Edge to develop webpages and web apps.
```

Comment:
```suggestion
Use the demos in the [MicrosoftEdge / Demos](https://github.com/MicrosoftEdge/Demos#readme) repo to learn how to use Microsoft Edge to develop extensions.
```

I'll leave that up to you, but this section of our TOC is specifically about developing browser extensions, not webpages or web apps. So this paragraph should reflect this.

Reply:
Use the extensions demos in the [Demos](https://github.com/MicrosoftEdge/Demos#readme) repo to learn how to develop extensions for Microsoft Edge.